tweet/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.urls import path
+from . import views
+
+app_name = 'tweet'
+
+urlpatterns = [
+  path('create', views.createTweet, name='createTweet'),
+  path('read/<int:id>', views.readTweet, name='readTweet'),
```

Comment:
Read about CRUD ?

Reply:
I know about crud. What happend in this?

Reply:
Is there a read in CRUD

1. Index
2. New
3. Create
4. Show
5. Edit 
6. Update 
7. Destroy 


No other action names

Reply:
we can use show. I used by crud - read, will change it.